compute/instances/reference-content/identify-devices.mdx

```diff
@@ -110,7 +112,7 @@ root@test-instance:~# lspci -d '::0200'
 ```
 
 The filter selects the Network controller device class/Ethernet controller device subclass. Three PCI devices are visible, which correspond to the public network device, and two VPC Private Network devices. By itself, the output of this command
-if not enough to distinguish between public and private networks, and can not distinguish between multiple private networks either: this simply confirms their existence in the PCI hierarchy of the Instance. 
+is not enough to distinguish between public and private networks, and can not distinguish between multiple private networks either: this simply confirms their existence in the PCI hierarchy of the Instance.
```

Comment:
Why is the sentence cut? Shouldn't the "is not enough" part be next to "this command"?

Reply:
I think I did that to avoid a long line, since in markdown-like languages and empty line is usually needed to get a newline. But yes, it's pretty arbitrary and could just be a single line here.

Reply:
Ok got it! Maybe we can split the two sentences by skipping the line from "By itself" so the paragraph isn't too crowded!